ui/tests/integration/helpers/changelog-url-for-test.js

```diff
@@ -27,7 +27,7 @@ module('Integration | Helper | changelog-url-for', function (hooks) {
   });
 
   test('it builds the url for double-digit versions', function (assert) {
-    const result = changelogUrlFor(['1.13.0+ent']);
+    const result = changelogUrlFor(['1.13.0']);
     assert.strictEqual(result, CHANGELOG_URL.concat('1130'));
```

Comment:
Don't we use dashes for ours? https://openbao.org/docs/release-notes/2-0-0/

Reply:
We did in the Alpha/Beta versions :thinking: Not sure yet if I want to test it's working properly for those versions

Reply:
I'll leave it as is for now